plugins/modules/system/sefcontext.py

```diff
@@ -22,6 +22,13 @@
     type: str
     required: yes
     aliases: [ path ]
+  equiv:
+    description:
+    - set target selinux equivalence to this path.
+    - setype must be set, the bogus 'many_t' works.
+    - when set, ftype, seuser, selevel are ignored.
+    type: str
+    required: no
```

Comment:
Use `true`/`false` for YAML booleans. (Check out the current `main` branch, it should do that for all options in this module.)
```suggestion
    required: false
```